e2e-tests/playwright/e2e/learning-path-page.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { Common } from '../utils/Common';
+
+test.describe('Learning Paths', () => {
+  let common: Common;
+  let uiHelper: UIhelper;
+
+  test.beforeEach(async ({ page }) => {
+    uiHelper = new UIhelper(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify that links in Learning Paths for Backstage opens in a new tab', async ({
+    page,
+  }) => {
+    await uiHelper.openSidebar('Learning Paths');
+
+    for (let i = 0; i < 5; i++) {
+      const popupPromise = page.waitForEvent('popup');
+      await page.locator(`div[class*="MuiCardHeader-root"]`).nth(i).click();
+      const popup = await popupPromise;
+      await popup.waitForLoadState();
+      console.log(await popup.evaluate('location.href'));
```

Comment:
@teknaS47 Instead of this console log can we do some assertion that new tab didn't open a blank tab?

Reply:
@subhashkhileri  I have pushed a new commit with the requested requirement, let me know if there is anything else, thanks!